### DISCUSSION TITLE
I can't register with gmail

### DISCUSSION THREAD

**Discussion by AzureVDT** [vercel/community#3853] [Account Issue] [answered]
### Question

I registered an account on Vercel using my Gmail, and I had successfully completed the initial step of confirming my Gmail. Unfortunately, my house experienced a power outage, which prevented me from completing the subsequent steps to finalize the account registration. Now, when I try to reattempt the registration, I find that I'm unable to sign up or log in using Gmail, even though I had previously completed the Gmail confirmation step with Vercel. Please assist me in resolving this issue.

**Comment by amyegan** [comment]
Do you see an error message when you try to log in at https://vercel.com/login?

**Reply by AzureVDT** [reply]
yah, i see the error message is We are unable to complete your signup. Please read [this guide], But when i click on the this guide link, i don't see any guide to complete next step, i trying to sign up again with that email but it alert The email address already exists.

**Reply by amyegan** [reply, marked as answer]
If you already created an account, you should use the [log in](https://vercel.com/login) page instead of the [sign up](https://vercel.com/signup) page.
